migrate/20230807_update_billing.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    DB[:billing_rate].truncate(cascade: true)
+    copy_into :billing_rate, data: <<COPY
+139d9a67-8182-8578-a303-235cabd5161c	VmCores	standard	hetzner-fsn1	0.000171296
+08c502f7-df5d-8978-9896-feafa0ec5c40	VmCores	standard	hetzner-hel1	0.000154167
+COPY
+  end
+end
```

Comment:
@byucesoy thought you might appreciate that my approach to this is kinda lame, but lucid, in that we see the target state (rather than adding and deleting rows, which is prone to additive error)

Reply:
Billing records have foreign key to billing rates, so TRUNCATE CASCADE would delete billing records as well. I guess similar thing can be done using TRUNCATE without CASCADE and using DEFERRED constraints.

Or we can remove the foreign key constraint, or I can move billing rates to a Ruby array (or JSON/YAML file), which I'm inclining towards to as we find these issues. In previous life, billing rates were Ruby array, which worked fine. This time I preferred keeping them in DB because I expect its size to be in the order of 10k, which is big but still manageable with arrays.

Reply:
I think we might need a modified programming model. Here's another puzzle for you: rate changes. they may need to be versioned. But...should they all be in the operating table, or put somewhere else? Unknown....